### PR TITLE
FIX pSMAC

### PR DESCRIPTION
**Pull request by mlindauer**
* FIX default path for input_psmac_dirs
* FIX parsing of runhistory files; skip lock files
* ADD debug statement to see runhistory location in logs

### CODE REVIEW

File: smac/optimizer/pSMAC.py [hunk 1]
```diff
@@ -47,6 +47,7 @@ def read(run_history: RunHistory,
             if match:
                 runhistory_file = os.path.join(output_directory,
                                                file_in_output_directory)
+                print(runhistory_file)
```
Comment (mfeurer) [inline smac/optimizer/pSMAC.py]: We shouldn't have any print statements in our code.

Comment (mlindauer) [reply smac/optimizer/pSMAC.py]: Sorry, my mistake. fixed now.

**Comment by mfeurer** [comment]
Test are still failing:
```======================================================================
ERROR: test_write (test.test_smbo.test_pSMAC.TestPSMAC)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/automl/SMAC3/test/test_smbo/test_pSMAC.py", line 78, in test_write
    pSMAC.write(run_history, self.tmp_dir)
TypeError: write() missing 1 required positional argument: 'logger'
```

**Comment by mfeurer** [comment]
Thanks for fixing the tests, but now there are still some files left by the tests:
```
OK
ERROR: tests did not clean up all files they generated:
  test/test_files/test_json_origin.json.lock
  test/test_files/validation/validated_runhistory.json.lock
  test/test_files/validation/validated_runhistory_EPM.json.lock
```

**Pull request merged**